components/p3a/network_annotations.cc

```diff
@@ -0,0 +1,165 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/p3a/network_annotations.h"
+
+#include "brave/components/p3a/brave_p3a_uploader.h"
+
+namespace brave {
+
+net::NetworkTrafficAnnotationTag GetRandomnessRequestAnnotation() {
+  return net::DefineNetworkTrafficAnnotation("p3a_star_randomness", R"(
+    semantics {
+      sender: "Brave Privacy-Preserving Product Analytics STAR Randomness Request"
+      description:
+        "Requests randomness for a single analytics metric."
+        "The randomness data is used to create a key for encrypting analytics data "
+        "using the STAR protocol, to protect user anonymity."
+        "See https://arxiv.org/abs/2109.10074 for more information."
+      trigger:
+        "Requests are automatically sent at intervals "
+        "while Brave is running."
+      data: "Anonymous usage data."
+      destination: WEBSITE
```

Comment:
I don't really follow why all our traffic annotations specify WEBSITE across the codebase, but this is clearly wrong. We probably need to introduce our own destination
https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/privacy/traffic_annotation.proto;drc=479d874a7fb07dd780fa8dfe5408106717398f8c;l=107

Reply:
ok if we discuss in another issue? seems like this may require some discussion?

Reply:
sure thing